.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: required
+language: rust
+rust:
+  - nightly
+  - beta
+  - stable
+
+env:
+  - GTK=3.4
+  - GTK=3.18
+
+addons:
+  apt:
+    packages:
+      - libcurl4-openssl-dev
+      - libelf-dev
+      - libdw-dev
+      - binutils-dev
+      - libgtk-3-dev
+      - libsoup2.4-dev
+      - libxtst-dev
+      - at-spi2-core
+      - openbox
```

Comment:
Not sure why libsoup and libcurl are needed, but ok :)

Reply:
Me neither. This travis manifest was just copied in full from gtk-test. If it's needed there, it's probably needed here as well. It's definitely needed for webkit2gtk, but I would do those parts in a separate repo anyway...